wasm/index.html

```diff
@@ -32,6 +42,29 @@
     <h1>
       <img src="logo.png" alt="Gother World">
     </h1>
+    <p>
+      <a class="badge" href="#" onclick="shutdown()">STOP</a>
+      <a class="badge" href="#" onclick="handleKeyEvent('l', 76, 'keydown'); setTimeout(() => handleKeyEvent('l', 76, 'keyup'), 50);">LOAD</a>
+      <a class="badge" href="#" onclick="handleKeyEvent('s', 83, 'keydown'); setTimeout(() => handleKeyEvent('s', 83, 'keyup'), 50);">SAVE</a>
```

Comment:
minor magic numbers used for the timeout

Reply:
yeah should be fixed with #45 as well